libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -363,36 +423,36 @@ void AP_Frsky_Telem::send_D(void)
     // send frame1 every 200ms
     if (now - _D.last_200ms_frame >= 200) {
         _D.last_200ms_frame = now;
-        send_uint16(DATA_ID_TEMP2, (uint16_t)(AP::gps().num_sats() * 10 + AP::gps().status())); // send GPS status and number of satellites as num_sats*10 + status (to fit into a uint8_t)
-        send_uint16(DATA_ID_TEMP1, gcs().custom_mode()); // send flight mode
-        send_uint16(DATA_ID_FUEL, (uint16_t)roundf(_battery.capacity_remaining_pct())); // send battery remaining
-        send_uint16(DATA_ID_VFAS, (uint16_t)roundf(_battery.voltage() * 10.0f)); // send battery voltage
+        send_uint16(_port, DATA_ID_TEMP2, (uint16_t)(AP::gps().num_sats() * 10 + AP::gps().status())); // send GPS status and number of satellites as num_sats*10 + status (to fit into a uint8_t)
```

Comment:
I don't really like hunting around for *which* `send_uint16` this is.

`using FRSky` to get that single symbol seems a little like overkill.  Perhaps have a `send_uint16_t method that does the `FRSky::send_uint16`?

... that's if we really do need to have FRSky as a stand-alone static class....

Reply:
I'll wrap it, just like the parse I'd like @ChrisBird to reuse as much as possible to send/recv frsky frames, no need to reinvent the weel